### DISCUSSION TITLE
How to reset collected wings

### DISCUSSION THREAD

**Discussion by Kieaer** [thatskymod/Sky-CotL-Scripts#111] [Q&A] [answered]
It's always the maximum wings even if eden run done
If i collect wings in the home of light, i will get about 30 wings, but when i get to the spawn point, i have 116 wings.
Drop wings and eden run again not working

**Comment by thatskymod** [comment, marked as answer]
It appears this question has been addressed before, though the comments seem to be missing.

To reiterate: Winged Lights collected from **Spirits** are **permanent** and cannot be reset. They may **seem to disappear** when you take damage from poisoned water, krill, or shards, but will eventually resurface once you are reborn. This is a technical quirk on TGC's end and not something that mods can access or fix.

LibSmite did a great job of warning users about this beforehand: https://github.com/thatskymod/Sky-CotL-Scripts/discussions/80#discussioncomment-9480710. Langit also recently implemented a similar warning (attached image).

In short, what you're describing (dropping wings) isn't possible. I apologize for the inconvenience, but I hope this clarifies things.

![WL_Permanent_Warning](https://github.com/user-attachments/assets/2e765a44-5472-47b6-bcfc-e20eff39322e)

**Reply by 2021em121-hue** [reply]
Tôi muốn bản mod này quay trở lại
